docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/index.md

```diff
@@ -52,7 +52,7 @@ Webhooks allow this third-party software to send their own internal event stream
 
 - [Setup your first webhook](/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/setup-webhooks/index.md)
 
-**Note: once you have setup a collector and tracker or webhook, you can pause and perform the remainder of the setup steps later**. That is because your data is being successfully generated and logged. When you eventually setup enrich[](#step-3-setup-enrich), you will be able to process all the data you have logged since setup.
+**Note: once you have setup a collector and tracker or webhook, you can pause and perform the remainder of the setup steps later**. That is because your data is being successfully generated and logged. When you eventually setup enrich, you will be able to process all the data you have logged since setup.
```

Comment:
This looks like a poorly formatted link which has been caught, although I think it it only anchors down about 2 lines of text (when it worked once upon a time, which I have doubts it did recently) so probably fine to remove it anyway.

Reply:
This was one of the few I changed by hand, exactly because the linked section was just below.